www/xkahj06/Projekt2/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="cs">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Get Reddit post</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Noto+Sans+TC" rel="stylesheet">
+</head>
+
+<body onload="LoadData()">
```

Comment:
tady nesmi byt inline js

Reply:
ah ok, mrknu jestli to jde dát někam do js...